src/main/java/org/infernalstudios/shieldexp/events/ShieldExpansionEvents.java

```diff
@@ -243,7 +245,15 @@ public boolean validateBlocking(Player player) {
     //reads a shield attribute from the given shield's stats map, or the default map if no map is found
     public static Double getShieldValue(Item item, String value) {
         String key = ForgeRegistries.ITEMS.getKey(item).toString();
-        return SHIELD_STATS.containsKey(key) ? SHIELD_STATS.get(key).get(value) : SHIELD_STATS.get("shieldexp:default").get(value);
+        Map<String, Double> statsMap = null;
+        if(SHIELD_STATS.containsKey(key))
+            statsMap = SHIELD_STATS.get(key);
+        if(statsMap == null)
+            SHIELD_STATS.get("shieldexp:default");
```

Comment:
You are not assigning this value to anything. What are you trying to do here?

Reply:
corrected the assignment 👍